charts/opentelemetry-ebpf/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: opentelemetry-ebpf
+version: 0.1.0
+description: OpenTelemetry eBPF Helm chart for Kubernetes
+type: application
+home: https://opentelemetry.io/
+sources:
+  - https://github.com/open-telemetry/opentelemetry-ebpf
+maintainers:
+  - name: dmitryax
+  - name: TylerHelmuth
```

Comment:
@nicolastakashi thank you for all the work! Do you want to help with this helm chart going forward and list yourself here as well?

Reply:
Oh sure, I'd love to help with this 😍